src/runmode-pfring.c

```diff
@@ -123,7 +123,11 @@ static void *OldParsePfringConfig(const char *iface)
         pfconf->threads = 1;
     } else {
         if (threadsstr != NULL) {
-            pfconf->threads = atoi(threadsstr);
+            if (ByteExtractStringUint16(&pfconf->threads, 10, 0, (const char *)threadsstr) < 0) {
+                SCLogWarning(SC_ERR_INVALID_VALUE, "Invalid value for pfconf "
```

Comment:
Suggest changing the message ... the invalid value comes from `pfring.threads` ... `pfconf threads` is correct but `pfring.threads` would be more helpful.